.circleci/xfail.txt

```diff
@@ -9,6 +9,11 @@ ETH.tut-7-rpc
 ALGO.tut-7-rpc
 CFX.tut-7-rpc
 
+# Interactive
```

Comment:
You can look at how `tut-8` works and I script an interactive program for testing.

Reply:
Alternatively, you could just make an interactive mode for `examples/duoswap/index.mjs`

Reply:
At the very least, you should have a README explaining how to test interactively and what to expect to see

Reply:
Made an option to run interactively in the original `duoswap` directly. The test will just run the automated version. Did a Readme, too

Reply:
Can the README have a sample interactive session log?